assets/src/js/_acf-experiments.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Experiments TypeScript
+ *
+ * @package    Secure Custom Fields
+ * @subpackage Admin
+ * @since      6.4.2
+ */
+
+// Define interfaces for our data structures
+interface ExperimentsData {
+  [key: string]: boolean; // Just store enabled status
+}
+
+interface ScfExperiments {
+  data: ExperimentsData;
+}
+
+// Declare global types
+declare global {
+  interface Window {
+    scfExperiments?: ScfExperiments;
+    SCF?: {
+      experiments?: {
+        data: ExperimentsData;
+        init: () => void;
+        isEnabled: (name: string) => boolean;
+      };
+    };
+  }
+}
+
+// Create a module to avoid global scope augmentation issues
+export {};
+
+// Initialize the SCF namespace if it doesn't exist
+if (typeof window.SCF === 'undefined') {
```

Comment:
Shouldn't we reuse the existing `acf` namespace until we migrate it?

Reply:
Yep. You are right. Otherwise it will be difficult to differentiate which is inside of each object.

Reply:
Done!